mongodb/install.sls

```diff
@@ -0,0 +1,18 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+add_mongodb_package_repository:
+  pkgrepo.managed:
+    - name: {{ mongodb.repo }}
+    - humanname: MongoDB 2.6.5 Repository
+    - dist: {{ mongodb.repo_dist }}
+    - keyid: {{ mongodb.key }}
+    - keyserver: {{ mongodb.keyserver }}
+    - refresh_db: True
```

Comment:
I would argue that this should either be configurable or moved into a separate file so that users have the option of adding it to their top file. If it's a separate file you could still include it in `init.sls` by default, but that makes it opt in instead of requiring hackery to stop it from being executed. Some people might want to just use the distro provided version.